templates/mysql/0/docker-compose.yml.tpl

```diff
@@ -0,0 +1,42 @@
+version: '2'
+services:
+  mysql-lb:
+    image: rancher/lb-service-haproxy
+    links:
+      - mysql
+    ports:
+      - ${mysql_lb_port}:3306
```

Comment:
- Thanks to add lb version.
- Link for semantics is ok. :)
- Lb doesn't work like that if user change the port. Here you need to define just the public port, not the backend one. The backend port is defined as  ```target_port: 3306``` in rancher-compose.yml . Please, change it  as
```
    ports:
-      - ${mysql_lb_port}:3306
+      - ${mysql_lb_port}:${mysql_lb_port}
```